<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI</class>
 <widget class="QMainWindow" name="UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxTyp">
      <item>
       <property name="text">
        <string>Standard</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Premium</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonListe">
      <property name="text">
       <string>Liegenliste anzeigen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBuchen">
      <property name="text">
       <string>Liege buchen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonStornieren">
      <property name="text">
       <string>Buchung stornieren</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelListe">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
